2022/src/main/java/frc/robot/subsystems/IntakeSubsystem.java

```diff
@@ -0,0 +1,33 @@
+package frc.robot.subsystems;
+
+import com.ctre.phoenix.motorcontrol.ControlMode;
+import com.ctre.phoenix.motorcontrol.NeutralMode;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.DriveUtil;
+import static frc.robot.Constants.*;
+
+public class IntakeSubsystem extends SubsystemBase {
+
+    private final WPI_TalonSRX m_intakeTalon;
+
+    public DrivetrainSubsystem() {
+        m_intakeLeft = new WPI_TalonSRX(Ports.CAN_INTAKE_LEFT_TALONSRX);
+        m_intakeRight = new WPI_TalonSRX(Ports.CAN_INTAKE_RIGHT_TALONSRX);
+        configureMotors();
+        m_intakeLeft.setSelectedSensorPosition(0);
```

Comment:
We can remove the `setSelectedSensorPosition` for both motors because we don't care about PID on intake